.travis.yml

```diff
@@ -36,6 +36,24 @@ matrix:
       env: PYTHON_VERSION=3.6 PIP_DEPENDENCIES='git+https://github.com/sphinx-doc/sphinx.git#egg=sphinx coveralls'
            CONDA_DEPENDENCIES="setuptools cython numpy pytest-cov"
 
+    # Test conda's clang
+    - os: osx
+      env:
+        - PYTHON_VERSION=3.5
+        - CONDA_DEPENDENCIES="setuptools sphinx cython numpy pytest-cov clang llvm-openmp"
+      before_install:
```

Comment:
I'm pretty sure there is no need for these `before_install` sections either, put those variable into the `env` section.

Reply:
So, I added these here following the Travis documentation. Also, because of the generic `before_install`, which then has to be overloaded per explicit job.

Reply:
_c.f._ https://github.com/astropy/astropy-helpers/pull/382/files#r191006134